eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -1,5 +1,6 @@
 parameters:
   Directory: 'not-specified'
+  IgnoreLinksFile: "$(Build.SourcesDirectory)/eng/ignore-links.txt"
```

Comment:
We shouldn't need to make these changes manually. Lets try to get the sync PR merged https://github.com/Azure/azure-sdk-for-go/pull/11024/files.

Reply:
Yes, will rebase my changes once the sync PR gets in. @KarishmaGhiya is working on merging the sync PR.